docs/documentation/jetstream/manage.md

```diff
@@ -44,3 +44,13 @@ Unlike streams, consumers are accessed by NATS client libraries as part of messa
 
 [!code-csharp[](../../../tests/NATS.Net.DocsExamples/JetStream/ManagingPage.cs#consumer-create)]
 [!code-csharp[](../../../tests/NATS.Net.DocsExamples/JetStream/ManagingPage.cs#consumer-get)]
+
+### Ephemeral and Durable Consumers
+
+A consumer can be ephemeral or durable. It is considered durable when an explicit name is set on the
+DurableName field when creating the consumer, otherwise it is considered ephemeral. Durables and ephemeral behave
+exactly the same except that an ephemeral will be automatically cleaned up (deleted) after a period of inactivity,
+specifically when there are no subscriptions bound to the consumer.
```

Comment:
https://natsbyexample.com/examples/jetstream/pull-consumer/go states that:

> ephemerals do not support multiple subscribers

Which would be good to note as a difference, if it is in-fact true

Reply:
I tried using two consume calls concurrently and that worked as expected. Not sure if it's not a limitation anymore or I'm missing a trick. @wallyqs do you know?

Reply:
Sounds like it might be a limitation on push consumers which the clients / new JS api doesn't support publicly any more.

Reply:
That's good info then. Maybe NATS By Example is out-of-date too.

Reply:
proposed a fix for nats-by-example: https://github.com/ConnectEverything/nats-by-example/pull/176